tests/api_tests/user_tests.py

```diff
@@ -0,0 +1,27 @@
+from nose.tools import assert_true, assert_false, eq_
+
+from app import app
+
+# Here's our "unit tests".
+class TestUsers():
+
+	def setUp(self):
+		self.app = app.test_client()
+
+	def tearDown(self):
+		pass
+
+	def test_get(self):
+		rv = self.app.get('/api/users')
+		eq_(len(rv.data),355) # size of sample input text
+
+	def test_post(self):
```

Comment:
More tests! There should be ones to make sure the required fields are there. What should happen if email isn't present?

What about puts? gets?